src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
In **navigation__items-list__item**, the element **item** is nested within the element **items-list**.  BEM says we can't nest elements inside other elements, even if they are nested in HTML. In this case, you have 2 possible options:

1. Rename it to **navigation__item**
2. Create a separate **items-list** block that will contain the **items-list__item** element inside

If you will be using **navigation__items-list** outside of navigation, then go with option 2, but I'm sure you won't be using it outside of navigation, so just go with option 1